src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/DeleteUserProfileFuncTest.java

```diff
@@ -21,6 +23,7 @@
 
 
 @RunWith(SpringIntegrationSerenityRunner.class)
+@WithTags({@WithTag("testType:Functional")})
```

Comment:
try to add tag in the super class

Reply:
already tried, it doesnt work

Reply:
following two tags needs to be together , then only tagging works, also if put it in super class then it requires at least 1 test case in that class.
@RunWith(SpringIntegrationSerenityRunner.class)
@WithTags({@WithTag("testType:Integration")})

Reply:
> already tried, it doesnt work

it is working tested in the PRD and please follow same in the UP